### PR TITLE
Updating other notices, I think?

### PR DESCRIPTION
**Pull request by leios**

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -266,15 +266,19 @@
 		<summary>
 			<div class="text-xl font-medium">Can I use copyrighted material?</div>
 		</summary>
-		<p>By registering as a creator you agree to the following copyright notice:</p>
```
Comment (fcrozatier) [inline src/routes/+page.svelte:269]: Shouldn't we keep this paragraph?

Comment (leios) [reply src/routes/+page.svelte:269]:
```suggestion
		<p>By registering as a creator you agree to the following copyright notice:</p>
```

Comment (leios) [reply src/routes/+page.svelte:269]: oops

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -266,15 +266,19 @@
 		<summary>
 			<div class="text-xl font-medium">Can I use copyrighted material?</div>
 		</summary>
```
Comment (leios) [inline src/routes/+page.svelte:268]:
```suggestion
		</summary>
		<p>By registering as a creator you agree to the following copyright notice:</p>
```

File: src/routes/+page.svelte (hunk 1 above)
Comment (leios) [reply src/routes/+page.svelte:269]: ok, fixed

**Pull request merged**